server/config/index.js

```diff
@@ -15,16 +17,19 @@ if (process.env.GRAPHIQL_QUERY_FILE) {
   }
 }
 
+const port = process.env.HTTP_PORT || '8000'
+
 const config = {
   server: {
-    port: process.env.HTTP_PORT || '8000'
+    port
   },
   graphQLConfig: {
     tracing: true
   },
   graphiqlConfig: {
     endpointURL: '/graphql', // if you want GraphiQL enabled
-    query: graphiqlQueryFileContent
+    query: graphiqlQueryFileContent,
+    subscriptionsEndpoint: `ws://${hostname()}:${port}/subscriptions`
   },
   postgresConfig: {
     database: process.env.POSTGRES_DATABASE || 'aerogear_data_sync_db',
```

Comment:
IHMO the fixed values as `'aerogear_data_sync_db'` could be declared as const on to of it.

Reply:
+1 on that. @camilamacedo86 great suggestion. I will create a GH issue for that.
I don't want this PR to get any bigger.

Reply:
https://github.com/aerogear/data-sync-server/issues/33